GoRest.Api.Tests/Comments/Create/CreateCommentPositiveTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using API_Tests.Asserts;
+using API_Tests.Helpers;
+using GoRest.Api.Client.Client;
+using GoRest.Api.Client.Client.Builder;
+using GoRest.Api.Client.Client.Interfaces.Controllers;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+
```

Comment:
delete net needed empty lines everywhere in this PR

Reply:
Done